code/__DEFINES/economy.dm

```diff
@@ -27,4 +27,6 @@
 #define ACCOUNT_SEC "SEC"
 #define ACCOUNT_SEC_NAME "Defense Budget"
 
-#define NO_FREEBIES "commies go home"
\ No newline at end of file
+#define NO_FREEBIES "commies go home"
+
+#define MAX_SECRUITY_CITATION 1000
```

Comment:
SECURITY not SECRUITY

Reply:
Friggin Atom just straight up remembers the typo ugh.